src/pages/chat/chat.tsx

```diff
@@ -67,32 +134,43 @@ export default function Chat() {
 
     const isLoadingPrompt = validLoadingPrompts.includes(lowerCasePrompt);
 
-    // The loading state will be shown for 4 seconds for loading prompt and 1.5 seconds for rest of the prompts
-    const waitTimeBeforeResponse = isLoadingPrompt ? 4000 : 1500;
-
     // Send Gen-AI response, replacing the loading chat bubble
+
     setTimeout(() => {
-      const validPrompt = VALID_PROMPTS.find(({ prompt }) => prompt.includes(lowerCasePrompt));
+      const validPrompt =
+        fileValue.length > 0
+          ? VALID_PROMPTS.find(({ prompt }) => prompt.includes('file'))
+          : VALID_PROMPTS.find(({ prompt }) => prompt.includes(lowerCasePrompt));
 
       setMessages(prevMessages => {
-        const response = validPrompt ? validPrompt.getResponse() : getInvalidPromptResponse();
+        const response = validPrompt ? validPrompt.getResponse(onSupportPromptClick) : getInvalidPromptResponse();
+
         prevMessages.splice(prevMessages.length - 1, 1, response);
         return prevMessages;
       });
       setIsGenAiResponseLoading(false);
-    }, waitTimeBeforeResponse + waitTimeBeforeLoading);
+      fileValue = [];
+    }, waitTimeBeforeResponse(isLoadingPrompt) + waitTimeBeforeLoading);
   };
 
   return (
     <div className={`chat-container ${!isVisualRefresh && 'classic'}`}>
       {showAlert && (
         <Alert dismissible statusIconAriaLabel="Info" onDismiss={() => setShowAlert(false)}>
-          This demo showcases how to use generative AI components to build a generative AI chat. The interactions and
-          functionality are limited.
-          <div>
-            1. To see how an incoming response from generative AI is displayed, ask "Show a loading state example".
-          </div>
-          <div>2. To see an error alert that appears when something goes wrong, ask "Show an error state example".</div>
+          <ExpandableSection
+            variant="inline"
+            headerText="This demo showcases how to use generative AI components to build a generative AI chat. The interactions and
+          functionality are limited."
+          >
+            <div>
+              1. To see how an incoming response from generative AI is displayed, ask "Show a loading state example".
+            </div>
+            <div>
+              2. To see an error alert that appears when something goes wrong, ask "Show an error state example".
+            </div>
+            <div>3. To see a how a file upload is displayed, upload one or more files.</div>
+            <div>4. To see support prompts, ask "Show support prompts".</div>
```

Comment:
Let's make this list an `<ol>` for improved accessibility:

```suggestion
      <TextContent>
        <ol>
          <li>
            To see how an incoming response from generative AI is displayed, ask
            "Show a loading state example".
          </li>
          <li>
            To see an error alert that appears when something goes wrong, ask "Show
            an error state example".
          </li>
          <li>
            To see a how a file upload is displayed, upload one or more files.
          </li>
          <li>To see support prompts, ask "Show support prompts".</li>
        </ol>
      </TextContent>


```